TeamPage.html

```diff
@@ -200,6 +200,48 @@ <h2>Meet Our Team</h2><br>
                     </div>
                 </div>
             </div>
+            <div class="col-sm-3 col-xs" style="min-width: 210px;">
+              <div class="id_presentation">
+                <img class="img_staff" src="https://i.imgur.com/DXgnG5Y.jpg" alt="Brandon's Picture"/>
+                 <div class="text_pres">Brandon Bae<br>
+                <div class="subtext_pres">Student Programmer</div>
+                 </div>
+                 <div class="container" style="text-align: center;">
+                  <button type="button" style="margin-top: 10px;" class="btn btn-danger btn-sm" data-toggle="modal" data-target="#myModalHelenB" onclick="openBSModal('myModalJungB', this)">Brandon's Profile</button>
+                  <div class="modal fade" tabindex="-1" id="myModalHelenB" role="dialog" datim-modal>
+                    <div class="modal-dialog">
+                      <div class="modal-content">
+                        <div class="modal-header">
+                          <div class="modal-title"><p class="text-justify"><b>Brandon Bae</b> - Student Programmer<br>
+                            Work Location: Remote<br>
+                            Email: jung.bae@unlv.edu</p>
+                          </div>
+                          <button type="button" class="close" data-dismiss="modal" aria-label="close">x</button>
+                        </div>
+                        <div class="modal-body">
+                          <p class="text-justify">Brandon is currently in his last year at UNLV pursuing a Bachelors of Science in Computer Science. Joining the UNLV-FIA group in January 2023 as a student programmer, Brandon will be working on remediating Section 508 violations. Besides work and school, Brandon enjoys relaxing by playing video games with friends, reading, or brushening up on his knowledge with some coding challenges </p>

```

Comment:
Two small things here:
1. "brushening up" should instead be "brushing up"
2. To match the other bios, can you end with a period on that last sentence?